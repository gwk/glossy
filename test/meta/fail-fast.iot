{
  'cmd': '$PROJ/iotest.py -no-times -fail-fast test-meta/fail-fast.iot',
  'links' : {'test-meta' : 'test-meta'},
}
