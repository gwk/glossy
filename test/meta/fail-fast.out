test-meta/fail-fast
test process was expected to return code: 0; actual code: 1.
test command failed: `false`

================================================================

